<launch version="0.1.1">
  <let name="bus_config_path" value="$(find-pkg-share nanotec_driver)/config/nanotec_cl3"/>
  <include file="$(find-pkg-share canopen_core)/launch/canopen.launch.py">
    <arg name="master_config" value="$(var bus_config_path)/master.dcf"/>
    <arg name="master_bin" value="$(var bus_config_path)/master.bin"/>
    <arg name="bus_config" value="$(var bus_config_path)/bus.yml"/>
    <arg name="can_interface_name" value="can0"/>
  </include>
</launch>